### DISCUSSION TITLE
Converting multi-vcf to maf

### DISCUSSION THREAD

**Discussion by xiw588** [PoisonAlien/maftools#738] [Q&A]
**Describe the issue**
I want to apply maftools to my multi-vcf file. Do you have any idea of doing this? 

Thank you.

**Command**
Please post your commands and the output (errors or any unexpected output)

```r
```

**Session info**
Run `sessionInfo()` and post the output below

```r
```

**Comment by PoisonAlien** [comment]
Hi,

You can use `vcftools` which has an option for splitting vcf files.
